controllers/rcj_soccer_referee_supervisor/recorder/recorder.py

```diff
@@ -0,0 +1,41 @@
+from controller import Supervisor
+
+
+class VideoRecordAssistant:
+
+    def __init__(self, supervisor, resolution="720p"):
+        self.supervisor = supervisor
+        self.resolution = resolution
+
+        if not isinstance(self.supervisor, Supervisor):
+            raise TypeError()
+
+    def create_title(self):
+        title = "test.mp4"
+        return title
+
+    def get_resolution(self):
+        res_table = {"480p": (720, 480),
+                     "720p": (1280, 720),
+                     "1080p": (1920, 1080)}
+
+        if(self.resolution not in res_table):
+            raise ValueError("Invalid Resolution")
+
+        return res_table[self.resolution]
+
+    def start_recording(self):
+        width, height = self.get_resolution()
+        filename = "./"+self.create_title()
+        self.supervisor.movieStartRecording(filename,
+                                            width,
+                                            height,
+                                            quality=100,
+                                            codec=0,
+                                            acceleration=1,
```

Comment:
@aquahika Is this something that we may be potentially interested in setting from say the command line or via an environment variable? I assume this sets the acceleration ratio and it feels like being able to set that may be of some use

Reply:
Yup, indeed.  I'll it be settable from external!